kibbeh/src/ui/BoxedIcon.tsx

```diff
@@ -0,0 +1,18 @@
+// React
+import React, { HTMLAttributes, ReactNode } from "react";
+
+interface BoxedIconProps extends HTMLAttributes<HTMLElement> {
+  icon: ReactNode;
+}
+
+export const BoxedIcon: React.FC<BoxedIconProps> = ({ icon, ...props }) => {
+  return (
+    <div
+      className="bg-primary-700 justify-center items-center rounded"
```

Comment:
Set the background to `gray-600` on hover and use `w-6 h-6` instead of inline styles https://github.com/benawad/dogehouse/blob/24d03500c4b4a84228cb897e9e30b8ac3304664d/kibbeh/tailwind.config.js#L42

Also `cursor-pointer`.